packages/vue-vuetify/src/complex/MixedRenderer.entry.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  isControl,
+  rankWith,
+  resolveSchema,
+  type JsonFormsRendererRegistryEntry,
+  type JsonSchema,
+  type Scopable,
+  type TesterContext,
+  type UISchemaElement,
+} from '@jsonforms/core';
+
+import isEmpty from 'lodash/isEmpty';
+import mixedRenderer from './MixedRenderer.vue';
+
+export const isMixedSchema = (
+  uischema: UISchemaElement & Scopable,
+  schema: JsonSchema,
+  context: TesterContext,
+) => {
+  if (!schema || typeof schema !== 'object') {
+    return false;
+  }
+
+  if (Array.isArray(schema.type)) {
+    return true;
+  }
+  if (schema.type === 'object') {
+    const schemaPath = uischema.scope;
+    if (schemaPath && !isEmpty(schemaPath)) {
+      let currentDataSchema = schema;
+      currentDataSchema = resolveSchema(
+        schema,
+        schemaPath,
+        context?.rootSchema,
+      );
+      if (currentDataSchema === undefined) {
+        return false;
+      }
+      if (Array.isArray(currentDataSchema.type)) {
+        return true;
+      }
+    }
+  }
+
+  return false;
+};
+
+export const isMixedControl = (
+  uischema: UISchemaElement,
+  schema: JsonSchema,
+  context: TesterContext,
+) =>
+  isMixedSchema(uischema, schema, context) &&
+  (isControl(uischema) || isDefaultGenUiSchema(uischema));
```

Comment:
Why do we check `isDefaultGenUiSchema`? Note also that we have adopters who adopt the default UI Schema, so this might not apply to them.

Reply:
it was needed because I wanted to make sure that besides the schema check we also check the uischema and the thing is that initially the code was
 `isMixedSchema(uischema, schema, context) && isControl(uischema)`
but then we weren't capturing the case where when the uischema is not provided by generated then the MixedRenderer was not involved - also it shouldn't affect others I guess since the mixed types are in general at the moment not really supported by the core / other renderers - I just didn't want the check to be just on the schema but to follow more what other testers were having as checks